src/GeositeFramework/region.json

```diff
@@ -11,20 +11,22 @@
     },
     "headerLinks": [
         { "text": "Azavea", "url": "http://www.azavea.com/" },
-        { "text": "GIS", "url": "http://en.wikipedia.org/wiki/Geographic_information_system" }
-    ],
-    "regionLinks": [
```

Comment:
Yeah -- one change in this update is that it moves what was previously in `regionLinks` into the `headerLinks`.

I'll update the schema & wiki and also update the New York & New Jersey develop region builds to match.